vMenu/Noclip.cs

```diff
@@ -17,14 +17,14 @@ public class NoClip : BaseScript
 
         private List<string> speeds = new List<string>()
         {
-            "Very Slow",
-            "Slow",
-            "Normal",
-            "Fast",
-            "Very Fast",
-            "Extremely Fast",
-            "Extremely Fast v2.0",
-            "Max Speed"
+            "Very Slow (1/8)",
+            "Slow (2/8)",
+            "Normal (3/8)",
+            "Fast (4/8)",
+            "Very Fast (5/8)",
+            "Extremely Fast (6/8)",
+            "Extremely Fast v2.0 (7/8)",
+            "Max Speed (8/8)"
```

Comment:
Added number values since RSM had number values and it's kinda nice to know what level of speed you're on.